app/modules/ads/slots-loader.js

```diff
@@ -0,0 +1,125 @@
+import {
+  context,
+  events,
+  eventService,
+  slotInjector,
+  slotRepeater,
+  utils,
+} from '@wikia/ad-engine';
+
+const logGroup = 'slots-loader';
+
+export const slotsLoader = {
+  baseSlotName: 'incontent_boxad',
+  incontentsCounter: 0,
+
+  getSlotId() {
+    return `${this.baseSlotName}_${this.incontentsCounter}`;
+  },
+
+  configureSlotsLoader() {
+    const bidsBackHandler = context.get('bidders.prebid.bidsRefreshing.bidsBackHandler');
+
+    if (!context.get('options.nonLazyIncontents.enabled')) {
+      return;
+    }
+
+    context.set(
+      'bidders.prebid.bidsRefreshing.bidsBackHandler',
+      () => {
+        const slotId = this.getSlotId();
+
+        utils.logger(logGroup, `refresh ended for: ${slotId}`);
+        context.get(`bidders.prebid.bidsRefreshing.${slotId}.resolve`)();
+        bidsBackHandler();
```

Comment:
I wish we could make a queue or an array on `bidders.prebid.bidsRefreshing.bidsBackHandler`, but it's a major :< For experiment purposes: it's fine.